src/pool/srv_pool_map.c

```diff
@@ -250,8 +256,8 @@ update_one_tgt(struct pool_map *map, struct pool_target *target,
 		case PO_COMP_ST_DOWN:
 		case PO_COMP_ST_NEW:
 			/* Nothing to do, already UPIN and DOWNOUT. DOWN can not be revert. */
-			D_INFO("Skip ADD_IN UPIN DOWN "DF_TARGET"\n",
-			       DP_TARGET(target));
+			D_INFO(DF_MAP ": Skip ADD_IN UPIN DOWN " DF_TARGET "\n",
```

Comment:
```suggestion
			D_INFO(DF_MAP ": Skip REVERT_REBUILD UPIN DOWN " DF_TARGET "\n",
```

Reply:
Changed to reflect the `case`s better. Please double check.